packages/api/src/routes/internal/medical/organization.ts

```diff
@@ -56,7 +57,7 @@ router.put(
       ? await updateOrganization({ id: orgDetails.id, ...organizationCreate })
       : await createOrganization(organizationCreate);
 
-    const syncInHie = isProvider(org);
+    const syncInHie = overrideProviderCheck || isProvider(org);
```

Comment:
nit: "override" infers using a provided value instead of another one, in this case it's more a "skip", no? Like, `skipProviderCheck`?